plugins/scaffolder-react/src/next/components/ScaffolderPageContextMenu/ScaffolderPageContextMenu.tsx

```diff
@@ -89,6 +92,20 @@ export function ScaffolderPageContextMenu(
         transformOrigin={{ vertical: 'top', horizontal: 'right' }}
       >
         <MenuList>
+          {onCreateClicked && (
+            <MenuItem onClick={onCreateClicked}>
+              <ListItemIcon>
+                <CreateComponentIcon fontSize="small" />
+              </ListItemIcon>
+              <ListItemText primary="Create" />
+            </MenuItem>
+          )}
+          {/* <MenuItem onClick={onCreateClicked}>
```

Comment:
Can we remove these comments? :pray:

Reply:
Oh, shoot! I was having some odd behaviour, fixed it and then clearly didn't clean up my code. Will do that later today!

Reply:
Fixed!